core/commands/log.go

```diff
@@ -35,8 +35,7 @@ var logLevelCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
 		Tagline: "Change the logging level.",
 		ShortDescription: `
-'ipfs log level' is a utility command used to change the logging
-output of a running daemon.
+Change the verbosity of one or all subsystems log output. Does NOT affect the event log.
```

Comment:
I would change `NOT` to not. It says it without needing to draw attention to it. Also, `This does not...` is better, I think.